arch/risc-v/src/qemu-rv32/Make.defs

```diff
@@ -20,7 +20,7 @@
 
 # Specify our HEAD assembly file.  This will be linked as
 # the first object file, so it will appear at address 0
-HEAD_ASRC = qemu_rv32_head.S
+HEAD_ASRC = qemu_rv32_head.S riscv_exception_common.S
```

Comment:
should we move to CMN_ASRCS? HEAD_ASRC can only contain one source file.

Reply:
Yes, we should do this, but some potential problem here, for FE310 on QEMU, it only works with `exception_common` implemented in fe310_head.S or disable the "C" extension.

Reply:
It seems that __trap_vec is not on 4 byte boundary in some situation.